app/src/main/java/org/simple/clinic/newentry/PatientEntryScreen.kt

```diff
@@ -284,7 +285,13 @@ class PatientEntryScreen(context: Context, attrs: AttributeSet) : RelativeLayout
   }
 
   override fun showColonyOrVillagesList(colonyOrVillageList: List<String>) {
+    ArrayAdapter<String>(context, android.R.layout.simple_list_item_1, colonyOrVillageList).also { adapter ->
```

Comment:
Let's create a layout file in our project instead of using the Android one, since we don't it will be the same across different versions or OEMs